### PR TITLE
ci(api-docs): show required changes with color

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

**Review by muniter — APPROVED** [review]

File: .github/workflows/api-docs.yml [hunk 1]
```diff
@@ -59,8 +59,7 @@ jobs:
         run: |
           echo "Job failed, run ./scripts/gen_vimdoc.py and commit your doc changes"
           echo "The doc generation produces the following changes:"
-          git --no-pager diff
-          exit 1
+          git diff --color --exit-code
```
Comment (jamessan) [inline .github/workflows/api-docs.yml:62]: `--no-pager` isn't needed?

Comment (dundargoc) [reply .github/workflows/api-docs.yml:62]: No. It's not interactive.

**Comment by justinmk** [comment]
btw, any reason `Remove reviewers` and `Request reviewers` are separate CI jobs? Seems like combining them could avoid some clutter.

**Comment by dundargoc** [comment]
They have dfferent triggers. Combining them is possible but makes the workflows more complicated (because you need to keep track of all possible triggers and use correct "if"s to separate each job properly. Can take a closer look tomorrow, but pretty sure it's not gonna be worth it.

**Comment by dundargoc** [comment]
You mean "clutter" as in the CI/jobs window in a PR?

**Pull request merged**